### PR TITLE
Tendermint URI RPC

### LINKED ISSUE #218: Expose Tendermint URI RPC

### PR DESCRIPTION
**Pull request by tzdybal**
This PR extends RPC functionality to enable URI RPC, for compatibility with Tendermint.

Resolves #218.

### CODE REVIEW

File: rpc/json/handler.go [hunk 1]
```diff
@@ -1,44 +1,65 @@
...
+		mux.HandleFunc("/"+name, h.newHandler(method))
 	}
+	return h
+}
+func (h *handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	h.m.ServeHTTP(w, r)
 }
 
-// ServeHTTP servces HTTP request
+// serveJSONRPC servces HTTP request
```
Comment (adlerjohn) [inline rpc/json/handler.go:45]: What's `servces`?

**Review by adlerjohn — COMMENTED** [review]
How can I test this?

File: rpc/json/handler.go (hunk 1 above)
Comment (tzdybal) [reply rpc/json/handler.go:45]: It's a typo.

**Comment by tzdybal** [comment]
Manual testing requires cosmos-sdk app to be running. I can update [demochain](https://github.com/tzdybal/demochain) if you want to test this manually.
We need to create end-to-end testing suite.

**Review by adlerjohn — APPROVED** [review]
Would be nice to have more instructions somewhere to know how to manually test these things. utACK

**Comment by tzdybal** [comment]
I assume "utACK" means "unit tested, ACK" ;)

**Comment by adlerjohn** [comment]
No, it means _untested_ ACK, which is about the same as unit tested 😂

**Pull request merged**